examples/pigweed-app/esp32/README.md

```diff
@@ -56,30 +56,41 @@ step. To install these components manually, follow these steps:
 Currently building in VSCode _and_ deploying from native is not supported, so
 make sure the IDF_PATH has been exported(See the manual setup steps above).
 
--   In the root of the example directory, sync the CHIP tree's submodules and
-    source `idf.sh`. Note: This does not have to be repeated for incremental
-    builds.
+-   Setting up the environment
 
-          $ source idf.sh
```

Comment:
should we delete idf.sh ? I believe all it does is to replace idf.py with idf

Reply:
@andy31415 In README we have focused upon CMake build system, and for CMake, we don't need idf.sh.

Reply:
@andy31415 Once all the examples support CMake and there are no teething issues, we can remove the auto make support. Till that time we will just mention CMake in the example READMEs. Is that fine?